userspace/engine/falco_engine.h

```diff
@@ -213,6 +214,15 @@ class falco_engine : public falco_common
 	bool is_plugin_compatible(const std::string &name, const std::string &version, std::string &required_version);
 
 private:
+	struct ruleset_node
+	{
+		ruleset_node(const std::string &n, falco_ruleset *p):
+			first(n), second(p) {}
+
+		// pair-like names
+		std::string first; // source
```

Comment:
Since we don't need `pair<>` anymore, wouldn't it be better to give these more significant names? This would make it more understandable for newcomer fellow contributors. My vote would go to name these `source` and `ruleset`. Let me know what you think!

Reply:
I was uncertain about it... let's do it :+1:

Reply:
... done :smile_cat: